packages/server/lib/browsers/webkit-automation.ts

```diff
@@ -0,0 +1,163 @@
+import _ from 'lodash'
+
+import type playwright from 'playwright-webkit'
+
+export type CyCookie = Pick<chrome.cookies.Cookie, 'name' | 'value' | 'expirationDate' | 'hostOnly' | 'domain' | 'path' | 'secure' | 'httpOnly'> & {
+  // use `undefined` instead of `unspecified`
+  sameSite?: 'no_restriction' | 'lax' | 'strict'
+}
+
+type CookieFilter = {
+  name: string
+  domain: string
+}
+
+function convertSameSiteExtensionToCypress (str: CyCookie['sameSite']): 'None' | 'Lax' | 'Strict' | undefined {
```

Comment:
```suggestion
const extensionMap = {
  'no_restriction': 'None',
  'lax': 'Lax',
  'strict': 'Strict',
} as const

function convertSameSiteExtensionToCypress (str: CyCookie['sameSite']): 'None' | 'Lax' | 'Strict' | undefined {
  return str ? extensionMap[str] : undefined
}
```

If you want to get rid of the `any`.

Reply:
Nice, TIL about `as const`. 9ac53fd8fad566d3cc0d4ecaa7ea59c33e66e631